src/Terrain/PlanetWrenderer.cpp

```diff
@@ -168,24 +163,6 @@ int IcoSphereTree::neighboor_index(SubTriangle& tri,
     return 255;
 }
 
-PlanetWrenderer::PlanetWrenderer()
-{
-    // Numbers that work nicely, though not ideal
-
-    m_chunkAreaThreshold = 0.04f;
```

Comment:
These are used as constants, so for the time being we can make them static constexpr. They can always go back to runtime variables if we need to make runtime changes, of course